app/models/manageiq/providers/openstack/network_manager/refresh_parser.rb

```diff
@@ -37,13 +37,11 @@ def ems_inv_to_hashes
       get_security_groups
       get_networks
       get_network_routers
-      get_floating_ips
       get_network_ports
+      get_floating_ips
```

Comment:
in case the order of the methods is really important, I'd write a comment to note that. Otherwise somebody comes around and wants to refactor this into alphabetical order :)